.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
     - ROS_REPO=ros
     - UPSTREAM_WORKSPACE=moveit.rosinstall
     - TEST_BLACKLIST=moveit_ros_perception  # mesh_filter_test fails due to broken Mesa OpenGL
+    - CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-unused-parameter -Wno-unused-but-set-parameter -Wno-unused-function"
```

Comment:
Maybe add a linebreak. And is there a reason why `-Wno-overloaded-virtual` is not set here?

Reply:
I'm not sure cmake can handle a linebreak in the CXXFLAGS variable.